debian/control

```diff
@@ -30,3 +30,22 @@ Description: Regolith customized gnome-flashback session
  This package contains the desktop and xsession configuration
  necessary to start a Regolith gnome-flashback / i3 desktop session.
  It is based on https://github.com/deuill/i3-gnome-flashback.
+
+Package: regolith-session-sway
+Architecture: any
+Depends: ${misc:Depends},
+    dbus # add more dependencies here,
+    sway,
+    swayidle,
+    xwayland,
+    clipman,
+    wl-clipboard,
+    #trawld,
```

Comment:
correctness: I believe this will fail to parse because due to the comment, there is nothing after `wl-clipboard` to read.  Either remove the comma on line 42 or enable the commented out lines.

Reply:
Actually I intend to have those lines enabled. But for now these packages don't really exist😅. I'll probably remove these for now and add them back once everything is packaged